libretroshare/src/gxs/rsgxsnetservice.cc

```diff
@@ -747,12 +747,12 @@ void RsGxsNetService::generic_sendItem(RsNxsItem *si)
 
 	RsGxsGroupId tmp_grpId;
 
-	if(mAllowDistSync && mGxsNetTunnel != NULL && mGxsNetTunnel->isDistantPeer( static_cast<RsGxsNetTunnelVirtualPeerId>(si->PeerId()),tmp_grpId))
```

Comment:
when I read this, it reminds me that mGxsNetTunnel is a pointer. That is why it is compared to a pointer. Implicily casting to bool for testing is far worse IMHO.

Reply:
<Joke>Could be rename to mpGxsNetTunnel and mbAllowDistSync.</joke> ;)
And we use it just after.
@G10h4ck Need to be clarified in Coding style.